app/src/main/java/de/danoeh/antennapod/dialog/FeedPreferenceSkipDialog.java

```diff
@@ -0,0 +1,40 @@
+package de.danoeh.antennapod.dialog;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import de.danoeh.antennapod.R;
+
+/**
+ * Displays a dialog with a username and password text field and an optional checkbox to save username and preferences.
+ */
+public abstract class FeedPreferenceSkipDialog extends AlertDialog.Builder {
+
+    public FeedPreferenceSkipDialog(Context context, int titleRes, int skipIntroInitialValue,
```

Comment:
Is there a case where the title might be different from `R.string.pref_feed_skip`? I do not think this needs to be a parameter.

Reply:
done